app/assets/stylesheets/components/_header.scss

```diff
@@ -0,0 +1,17 @@
+.sc-TakeoverHeader {
+
+  &-content {
+    @include display(flex);
+    @include justify-content(space-between);
+  }
+
+  &--multipleCta {
+    @include display(flex);
+
+    a {
+      font-size: 1.4rem;
+      text-transform: uppercase;
+      color: $white;
+    }
+  }
+}
```

Comment:
Summing up all of these `sc-TakeoverHeader` comments... I'm not sure we need to touch this component at all. Can we try extracting the link rules and deleting everything else?